src/diffusers/models/unet_2d_blocks.py

```diff
@@ -1897,7 +1914,7 @@ def __init__(
         resnet_act_fn: str = "swish",
         resnet_groups: int = 32,
         resnet_pre_norm: bool = True,
-        attn_num_head_channels=1,
```

Comment:
Similar to: https://github.com/huggingface/diffusers/pull/3797/files#r1234704127